extmod/vfs_posix_file.c

```diff
@@ -273,4 +280,49 @@ const mp_obj_vfs_posix_file_t mp_sys_stdin_obj = {{&mp_type_vfs_posix_textio}, S
 const mp_obj_vfs_posix_file_t mp_sys_stdout_obj = {{&mp_type_vfs_posix_textio}, STDOUT_FILENO};
 const mp_obj_vfs_posix_file_t mp_sys_stderr_obj = {{&mp_type_vfs_posix_textio}, STDERR_FILENO};
 
+#if MICROPY_PY_SYS_STDIO_BUFFER
+STATIC const mp_stream_p_t vfs_posix_buffer_stream_p = {
+    .read = vfs_posix_file_read,
+    .write = vfs_posix_file_write,
+    .ioctl = vfs_posix_file_ioctl,
+    .is_text = false,
+};
+MP_DEFINE_CONST_OBJ_TYPE(
+    mp_type_vfs_posix_buffer_inner,
+    MP_QSTR_TextIOWrapper,
```

Comment:
`sys_stdio.mphal.c` uses this name.

```suggestion
    MP_QSTR_FileIO,
```